consent/strategy_default_test.go

```diff
@@ -510,6 +510,345 @@ func TestStrategyLogout(t *testing.T) {
 	}
 }
 
+func TestStrategyLogoutPost(t *testing.T) {
```

Comment:
This looks quite similar to the other test, only with a different HTTP method. Could you refactor this to reduce the amount of code duplication? :)